cli/src/pcluster/models/hit/hit_cluster_model.py

```diff
@@ -72,7 +72,7 @@ def test_configuration(self, pcluster_config):
             else {}
         )
         try:
-            latest_alinux_ami_id = self._get_latest_alinux_ami_id()
+            cluster_ami_id = self._get_cluster_ami_id(pcluster_config)
```

Comment:
cluster_ami_id can be None if the query does not return any result. Shall we handle this?

Reply:
It should never happen in theory, but I agree. Fixed.